astronomer/airflow/version_check/models.py

```diff
@@ -88,5 +88,13 @@ class AstronomerAvailableVersion(Base):
     description = Column(Text)
     url = Column(Text)
     hidden_from_ui = Column(Boolean, default=False, nullable=False)
+    end_of_support = Column(UtcDateTime(timezone=True), nullable=True)
 
     __table_args__ = (Index('idx_astro_available_version_hidden', hidden_from_ui),)
+
+
+class DismissedEOLWarning(Base):
+    __tablename__ = "dismissed_eol_warning"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    version = Column(Text().with_variant(String(255), "mysql"), nullable=False)
+    dismissed_until = Column(UtcDateTime(timezone=True), nullable=False)
```

Comment:
Can't this just be another column on the primary table? "eos_dismissed_until"

Reply:
I have changed it.